src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static String SUPPLY_LABEL = "supply";
+    private static String BUY_LABEL = "buy";
+    private static String RESULT_LABEL = "result";
+    private static String DELIMITER = ",";
+    private static String NEW_LINE = "\n";
 
+    private int[] readAndAggregate(String fromFileName) {
+        int supply = 0;
+        int buy = 0;
+        int[] result = new int[2];
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
+            String[] key = line.split(",");
+            while (line != null) {
+                key = line.split(",");
+                if (key[0].equals("supply")) {
+                    result[0] += Integer.parseInt(key[1]);
+                    line = reader.readLine();
+                } else {
+                    result[1] += Integer.parseInt(key[1]);
```

Amount parsing uses Integer.parseInt(key[1]) without trimming and without catching NumberFormatException. If the amount is malformed this will throw an unchecked exception without a helpful message. Wrap parse errors and rethrow as a RuntimeException with the original exception as the cause and include the file name. This addresses checklist: "Don't ignore exceptions".